frontend/src/components/common/Header/Header.styles.ts

```diff
@@ -6,36 +6,85 @@ export const HeaderStyles = styled.header`
   left: 0;
   top: 0;
   width: 100%;
-  height: 60px;
+  height: 52px;
   z-index: 1;
 `;
 
+export const TextCoverStyles = styled.div`
+  display: flex;
+  width: 365px;
+`;
+
 export const LogoButtonStyles = styled.button`
-  flex-grow: 0;
-  border: 0;
+  margin-left: 130px;
+  width: 117px;
+  height: 41px;
+  background-color: transparent;
+  border: none;
   font-style: normal;
   font-weight: 400;
-  font-size: 36px;
-  height: 60px;
-  line-height: 60px;
+  font-size: 26px;
+  line-height: 40px;
   letter-spacing: -0.03em;
-  background-color: transparent;
   color: #000000;
-  margin-left: 140px;
   cursor: pointer;
 `;
 
 export const introduceButtonStyles = styled.button`
```

Comment:
```suggestion
export const introduceButtonStyles = styled.a`
```
"모아동 소개" 페이지 이동에는 button 대신 a 태그를 사용하는 방법도 있어요!

그렇게 하면 `background-color: transparent;`을 따로 설정할 필요 없구
따로 onClick 함수를 정의하지 않아도 href 속성만으로 다른 페이지로 이동할 수 있을 것 같아요😊

Reply:
a태그를 까먹고 있었네요 감사합니다 ㅎㅎ 

+ 대문자로 수정했습니다

Reply:
좋슴다👍👍